Cargo.toml

```diff
@@ -1,5 +1,6 @@
 [workspace]
 
 members = [
+  "nblast-py",
```

Comment:
I had excluded this because cargo shouldn't touch nblast-py. Maturin can build it fine, which is all we care about, and it only really needs to be tested from python. The build failure seems to be due to pynblast https://travis-ci.com/github/clbarnes/nblast-rs/jobs/324927202

Reply:
Unfortunate since RLS/vscode doesn't pick up nblast-py's rust stuff by default otherwise. Oh well, the commit is removed.

Reply:
Yeah not getting any coding help is a real pain - didn't realise that including it in the workspace would fix that. I guess it could be left in and just document that cargo should not be called from the workspace directory?